app/lib/registries/full_topic_taxonomy_registry.rb

```diff
@@ -6,6 +6,17 @@ def [](base_path)
       taxonomy[base_path]
     end
 
+    def find_parent_by_id id
```

Comment:
We can remove this once https://github.com/alphagov/finder-frontend/pull/1224 is merged.

Reply:
✅